package.json

```diff
@@ -47,15 +48,24 @@
     "productName": "Trakkr.",
     "npmRebuild": false,
     "files": [
-      "./dist/**/*",
       "./electron/**/*",
-      "index.html"
+      "index.html",
+      "dist/**/*",
+      "!dist/*-unpacked",
+      "!dist/*.AppImage",
+      "!dist/*.exe"
```

Comment:
Need to exclude some folders under dist from being included 
They aren't used during runtime, and caused the builds to be massive